packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -65,12 +65,11 @@ OpenID Connect is an authentication standard built on top of OAuth 2.0. It adds
 
 Although OpenID Connect is built on top of OAuth 2.0, the [OpenID Connect specification](https://openid.net/connect/) uses slightly different terms for the roles in the flows:
 
-* The "OpenID provider" &mdash; The authorization server that issues the ID token. In this case Okta is the OpenID provider.
-* The "end user" &mdash; Whose information is contained in the ID token
-* The "relying party" &mdash; The client application that requests the ID token from Okta
-
-* The "ID token" is issued by the OpenID Provider and contains information about the end user in the form of claims.
-* A "claim" is a piece of information about the end user.
+* **OpenID provider**: The authorization server that issues the ID token. In this case Okta is the OpenID provider.
+* **end user**: The end user's information that is contained in the ID token.
+* **relying party**: The client application that requests the ID token from Okta.
```

Comment:
```suggestion
* **relying party**: The client application that requests the ID token from Okta
```

Reply:
As discussed over Slack will stick with the period.